SU-SWS/stanford_components
# Earth 154 earth matters views

# (finally) READY FOR REVIEW

# Summary
- Simplified masonry blocks.

# Needed By (Date)
- launch

# Urgency
- high

# Steps to Test
1. composer update & sync.
2. Checkout this branch and `EARTH-151-earth-matters` branch of [stanford_news](https://github.com/stanford-earth/stanford_news/pull/6)
3. clear cache
4. `drush features-import stanford_news -y; drush en stanford_news_earth_matters -y`
5. go to /earth-matters page and make sure it works as designed. kinda like [this page](http://earthode43.prod.acquia-sites.com/earth-matters)

# Associated Issues and/or People
- https://github.com/stanford-earth/stanford_news/pull/6
- http://earthode43.prod.acquia-sites.com/earth-matters

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
